docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "2"
+x-services:
+  rails: &rails
+    build: .
+    environment:
+      PGHOST: "db"
+      PGUSER: "postgres"
+    env_file: 
+      - .env
+    volumes:
+      - bundle:/usr/local/bundle
```

Comment:
❓: Did the order here end up mattering in the end?

Reply:
Nope